ceph.spec.in

```diff
@@ -497,9 +497,13 @@ export RPM_OPT_FLAGS=`echo $RPM_OPT_FLAGS | sed -e 's/i386/i486/'`
 		--with-librocksdb-static=check \
 %if 0%{?rhel} || 0%{?fedora}
 		--with-systemd-libexec-dir=/usr/libexec/ceph \
+		--with-rgw-user=apache \
+		--with-rgw-group=apache \
```

Comment:
Please use `root` here.

The "apache" UID is not going to be available unless the httpd RPM is installed, and httpd is not a requirement for civetweb.